### DISCUSSION TITLE
cant get the multi script to work

### DISCUSSION THREAD

**Discussion by Bekloppten** [cmalf/getgrass-bot-js#17] [Q&A] [answered]
hi guys.  i started using 041 yesterday, and i think it works without any problems.
since then i tried to get the multi script to work.
![041 multi](https://github.com/user-attachments/assets/670945f1-b39e-450d-8b7d-d2b3834e87dc)
my account and uid files should be good. the login& token function of the multi script works without any problems. i updated the index files too, but nothing changed.
can somebody pls help me find my failure/problem ?

Thanks in Advance :D

**Comment by cmalf** [comment, marked as answer]
> hi guys. i started using 041 yesterday, and i think it works without any problems. since then i tried to get the multi script to work. ![041 multi](https://private-user-images.githubusercontent.com/163794379/398319815-670945f1-b39e-450d-8b7d-d2b3834e87dc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUwNTE1NjksIm5iZiI6MTczNTA1MTI2OSwicGF0aCI6Ii8xNjM3OTQzNzkvMzk4MzE5ODE1LTY3MDk0NWYxLWIzOWUtNDUwZC04YjdkLWQyYjM4MzRlODdkYy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIyNFQxNDQxMDlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wNTEyZDk0OGFjMGVlMDljYTBiNTdiMTA4MmUwMzBkZjUxNjcwYjU4NzVkNzE2ZTU4Y2Q2ZDVkNGE1ZTc3OTU4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.DQlbGk6HLd6oEb91cxvkmf9YJUxDjVYtuTat38UU2qU) my account and uid files should be good. the login& token function of the multi script works without any problems. i updated the index files too, but nothing changed. can somebody pls help me find my failure/problem ?
> 
> Thanks in Advance :D

have you read the note?
